pkgs/applications/misc/when/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, perl
+, installShellFiles
+, fetchFromBitbucket
+}:
+
+stdenv.mkDerivation {
```

Comment:
You could change `stdenv` to `stdenvNoCC` if you want to, 'cause this derivation is really just patching a perl script, no `gcc` is used